nucypher/cli/commands/ursula.py

```diff
@@ -402,6 +538,50 @@ def destroy(general_config, config_options, config_file, force):
     destroy_configuration(emitter, character_config=ursula_config, force=force)
 
 
+@ursula.command()
+@option_config_file
+@click.option(
+    "--keystore-filepath",
+    help="Path to keystore .priv file",
+    type=EXISTING_READABLE_FILE,
+)
+@click.option(
+    "--from-mnemonic",
+    help="View TACo public keys from mnemonic seed words",
+    is_flag=True,
+)
+def public_keys(config_file, keystore_filepath, from_mnemonic):
+    """Display the public keys of a keystore."""
+    emitter = StdoutEmitter()
+
+    if sum(1 for i in (keystore_filepath, config_file, from_mnemonic) if i) > 1:
```

Comment:
```suggestion
    if len(i for i in (keystore_filepath, config_file, from_mnemonic) if i is not None) != 1:
```

Reply:
- We have to be careful since `from_mnemonic` is a boolean (`False` by default), so we can't just check `is not None`.
- It is valid usage if none of the 3 arguments are set (with `from_mnemonic` being `False`), which is why `> 1` was used. In this case we use the default config file location which has a keystone path value in it.

@cygnusv , do you have a strong preference here between counting arguments and checking the length of the resulting list? If you indeed have a strong opinion, we could do the following
```python
len([i for i in (keystore_filepath, config_file, from_mnemonic) if bool(i)]) > 1
```

(open to any alternatives as well)

Reply:
> * it is valid usage if none of the 3 arguments are set (with `from_mnemonic` being `False`), which is why `> 1` was used. In this case we use the default config file location which has a keystore path value in it.

In that case, the wording of the error message above should be changed to `"At most one of --keystore-filepath, --config-file, or --from-mnemonic must be specified"`.

Reply:
👍 - addressed in https://github.com/nucypher/nucypher/pull/3545.